libpolys/polys/monomials/p_polys.cc

```diff
@@ -4529,8 +4529,8 @@ static inline unsigned long GetBitFields(const long e,
 // if (ev1 & ~ev2) then exp1 does not divide exp2
 unsigned long p_GetShortExpVector(const poly p, const ring r)
 {
-  assume(p != NULL);
   if (p == NULL) return 0;
```

Comment:
This line was added in this commit. I simply changed the order of assume and this line. 
f3094ae7 (Oleksandr Motsak  2012-07-23 17:29:18 +0200 4532)   if (p == NULL) return 0;

I will leave it as before my commit.